jobs/resource_job.go

```diff
@@ -218,7 +218,7 @@ type JobTaskSettings struct {
 	NewCluster        *clusters.Cluster   `json:"new_cluster,omitempty" tf:"group:cluster_type"`
 	JobClusterKey     string              `json:"job_cluster_key,omitempty" tf:"group:cluster_type"`
 	ComputeKey        string              `json:"compute_key,omitempty" tf:"group:cluster_type"`
-	Libraries         []libraries.Library `json:"libraries,omitempty" tf:"slice_set,alias:library"`
+	Libraries         []libraries.Library `json:"libraries,omitempty" tf:"slice,alias:library"`
```

Comment:
I don't think you need the `slice` here.

Just `tf:"alias:library"` should be enough

Reply:
by default arrays are converted into slices... `slice_set` was need if you don't want to depend on the order of elements